src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/FileImportService.cs

```diff
@@ -133,6 +139,7 @@ public async Task CheckComplete(DataImport import, StatisticsDbContext context)
                 if (import.Errors.Count == 0)
                 {
                     await _dataImportService.UpdateStatus(import.Id, COMPLETE, 100);
+                    await _dataImportService.WriteDataSetFileMeta(import.SubjectId);
```

Comment:
I have a feeling that the intention of the `CheckComplete` method was to set a final import state or handle errors.  I suggest either moving this to the code that calls `CheckComplete` and setting the metadata there, or renaming the method to `CompleteImport` or something along those lines.

Reply:
I've renamed it to `CompleteImport`